examples/workqueue/README.md

```diff
@@ -7,7 +7,7 @@ It demonstrates how to:
  * combine the workqueue with a cache to a full controller
  * synchronize the controller on startup
 
-The example is based on https://git.k8s.io/community/contributors/devel/controllers.md.
+The example is based on https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/controllers.md.
```

Comment:
Thanks for catching this!

If you could sign the CLA + keep the git.k8s.io link (instead of github.com), this lgtm. :+1:

Reply:
I should have already signed the CLA. Maybe it is e-mail matching based? I will try to open this PR using a different e-mail. Thanks for the review.

Reply:
> Maybe it is e-mail matching based

yeah, the CLA bot checks for the CLA as per the email :)